pages/index.js

```diff
@@ -2,15 +2,16 @@ import { Container, Tab, Row, Col, ListGroup } from "react-bootstrap";
 import { Layout } from "../reusable/layout/Layout";
 import { HomeSearch } from "../reusable/search/HomeSearch";
 
+import logo from '../assets/logo_transparent.png'
 import css from "./index.css";
 
 const Home = () => (
   <Layout>
     <Container className={css.homeContainer}>
       <Row className={css.homeBoxFullFlexWrapper}>
         <Col lg={12} className={css.homeLogoWrapper}>
-          <h1>JobHub</h1>
-          <p>Minimalist Jobs Platform. We love simplicity.</p>
+        <img className={css.logo} src={logo} alt="JobHub"/>
```

Comment:
Having in mind that we're using react-bootstrap, we can make use of the https://react-bootstrap.github.io/components/images/ components.

Reply:
Good, I'll make the changes in no time. 😉

Reply:
Done! ✔